hokusai/lib/common.py

```diff
@@ -36,11 +36,9 @@ def smart_str(s, newline_before=False, newline_after=False):
     s = '\n' + s
   if newline_after:
     s = s + '\n'
-  if isinstance(s, unicode):
```

Comment:
@ashkan18 this is where things start to get tricky.  In Python 2, the `str()` constructor is a bytestring, and `unicode` is a unicode string, but in Python 3, `str()` creates a unicode string and `bytes` creates a bytestring.  There are cases (see below) when you want to decode a bytestring to unicode, but in general everything should be unicode.  Do you remember why we made this smart string method and needed the statement...

```
if isinstance(s, unicode):
  return unicode(s).encode("utf-8")
```

I suppose we want to print some special UTF-8 characters so will need to refactor this method

Reply:
yeah! i actually don't remember why we had this `smart_str` method, but yes i can imagine we'd need UTF-8 support later.